### PR TITLE
Do file parts according to ZGW standard

### PR DESCRIPTION
**Pull request by rjzondervan**
File parts are created on mutations of enkelvoudigInformatieObject where a file size is set, but no content

### CODE REVIEW

File: Installation/Schema/zrc/zrc.objectIdentificatie.schema.json [hunk 1]
```diff
@@ -47,6 +47,93 @@
...
+      "maxLength": 80
+    },
+    "nummer": {
+      "type": "string",
+      "description": "Uniek identificerend administratienummer van een huishouden zoals toegekend door de gemeente waarin het huishouden woonachtig is.",
+      "title": "nummer",
+      "maxLength": 12
+    },
+    "isGehuisvestIn": {
+      "allOf": [
```
Comment (WilcoLouwerse) [inline Installation/Schema/zrc/zrc.objectIdentificatie.schema.json:100]: I don't think allOf does anything, currently

File: Service/DRCService.php [hunk 2]
```diff
@@ -331,21 +346,51 @@ public function inhoudHandler(array $data, array $configuration): array
...
+
+                for ($iterator = 0; $iterator < $parts; $iterator++) {
+                    $fileParts[] = $this->createFilePart($objectEntity, $iterator, ceil($data['bestandsomvang'] / $parts));
+                }
+
+                $objectEntity->hydrate(['bestandsdelen' => $fileParts]);
+
+                $this->entityManager->persist($objectEntity);
+                $this->entityManager->flush();
+                $this->entityManager->flush();
```
Comment (WilcoLouwerse) [inline Service/DRCService.php:385]: we still need 2 flushes? no better solution yet?

**Review by WilcoLouwerse — APPROVED** [review]

File: Service/DRCService.php [hunk 3]
```diff
@@ -310,10 +319,16 @@ private function generateDownloadEndpoint(string $id, Endpoint $downloadEndpoint
      */
     public function inhoudHandler(array $data, array $configuration): array
     {
+        var_dump('Hello darkness my old friend');
```
[the remark this answers is not in the document]
Comment (WilcoLouwerse) [reply Service/DRCService.php:322]: ^

**Review by WilcoLouwerse — DISMISSED** [review]
at least remove var_dump

File: Service/DRCService.php (hunk 3 above)
[the remark this answers is not in the document]
Comment (rjzondervan) [reply Service/DRCService.php:322]: Thanks! Will fix

**Review by smisidjan — APPROVED** [review]

**Review by bbrands02 — APPROVED** [review]

**Pull request merged**